tasks.py

```diff
@@ -10,7 +10,7 @@
 @task
 def pytest(c):
     c.run(
-        'pytest ./tests/unit ./tests/integration ./tests/performance/tests '
+        'pytest ./tests/unit ./tests/integration ./tests/performance/tests ./tests/datasets/tests '
```

Comment:
The tests here are just tests for the generators themselves correct?

Reply:
That's right